package.json

```diff
@@ -3,6 +3,9 @@
   "version": "1.1.14",
   "description": "Utility for detecting phishing domains targeting Ethereum users",
   "main": "src/index.js",
+  "engines": {
+    "node": ">=12.0.0"
```

Comment:
We might want to consider delaying this. This package is fairly important to keep up-to-date, so a breaking change like this could have a more severe impact than it would for most libraries, where the consequences of a delayed update are minor. This package is also used more widely than most of our others are.

Reply:
Makes sense. I've removed `engines` from `package.json` for now, and am specifying Node at 10 in the `.nvmrc` file added by this PR